### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader:NVDA is announcig normal table as clickable. 

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7607]
.
**GitHubTags**: #SQL Azure Data Studio;#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#DesktopApp;#NVDA;#Win32;
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727ff9fe68421b004bf270cd509e4c5de165
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Additional Details:**
NVDA 2019.2

**Steps to Reproduce :**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster” .
5.Start NVDA and Observe the announcement.
**Actual:**
NVDA is announcing the normal table as clickable as the table is not editable. 
**Expected:**
NVDA should not announce the table as clickable. 
**User Impact:**
Screen reader User will not able to understand for what purpose the check box is there.
**Attachment for Reference:**
[A11y_AsureDataStudio_Selectthedeploymentoption_Screenreader2.pptx](https://github.com/microsoft/azuredatastudio/files/3707565/A11y_AsureDataStudio_Selectthedeploymentoption_Screenreader2.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
@V-sheman  we have a similar issue and treated as exception due to the low impact of this behavior: https://github.com/microsoft/azuredatastudio/issues/6662, can we also close this issue?

@yualan , could you please help follow up on this?

**Comment by Charles-Gagnon** [comment]
Closing this as noted this was previously determined to be a low-impact issue and is something that would need to be fixed in SlickGrid anyways. Please reopen if there's a problem with this @V-sheman
